docs/resie_transient_effects.md

```diff
@@ -149,44 +153,32 @@ In the figure below, a part-load curve (orange curve) based on an exponential fu
 
 The basis for the consideration of non-linear part-load efficiencies is the user-defined efficiency curve of a component
 
-\(\eta(PLR)= f(PLR) \left (= \frac{E_{useful}(PLR)}{E_{expended}(PLR)} \right ) \)
+\(\eta(PLR) = \frac{E_{useful}(PLR)}{E_{expended}(PLR)}\)
 
-as shown exemplarily as orange curve in the figure above. \(f(PLR)\) can be a linear, an exponential or a partially defined function. The efficiency curve can also be a non-monotonic function as long as the function of the expended energy (yellow curve for electrical input in the figure above)
+as shown exemplarily as orange curve in the figure above. \(\eta(PLR)\) can be any continuous non-monotonic function as long as the function of the expended energy (yellow curve for electrical input in the figure above)
```

Comment:
The function _can_ be monotonic, but its not a requirement. This could be clarified, as the possibility of non-monotonic functions is a feature that is an advantage of the calculation methodology of ReSiE, but it's not a requirement.